common/Scripts/ModifyWindowsOptionalFeatures.cs

```diff
@@ -131,6 +147,24 @@ private static ExitCode FromExitCode(int exitCode)
         };
     }
 
+    private static ExitCode HandleProcessExecutionException(Exception ex, ILogger? log = null)
+    {
+        if (ex is System.ComponentModel.Win32Exception win32Exception)
+        {
+            if (win32Exception.NativeErrorCode == 1223)
+            {
+                log?.Information(ex, "UAC was cancelled by the user.");
+                return ExitCode.Cancelled;
+            }
```

Comment:
Should this log in the case for other error codes?

Reply:
To keep this change as simple as possible, I'll leave this as is. Can reevaluate error handling seperately.